src/server/api/circuits.go

```diff
@@ -95,6 +96,61 @@ func circuitLoadHandler(m interfaces.CircuitStorageInterfaceFactory, c *gin.Cont
 	c.JSON(http.StatusOK, circuit)
 }
 
+func circuitUpdateHandler(m interfaces.CircuitStorageInterfaceFactory, c *gin.Context, userId model.UserId) {
```

Comment:
Can you delete the "create" and "store" handler above now that this one function replaces them?